draft-ietf-quic-ack-frequency.md

```diff
@@ -327,13 +327,13 @@ send an acknowledgement immediately on receiving a reordered ack-eliciting
 packet. This extension modifies that behavior when an ACK_FREQUENCY frame with
 a Reordering Threshold value other than 1 has been received.
 
-Largest Unacked
+Largest Unacked:
```

Comment:
Maybe it'd be best to use a definition list style here?

Unless the rich diff is misleading, you now have two colons before the definition.